spec/index.bs

```diff
@@ -15320,10 +15336,11 @@ The [=texel block memory cost=] of each of these formats is the same as its
             <th><span class=vertical>multisampling</span>
             <th><span class=vertical>resolve</span>
             <th><span class=vertical>{{GPUTextureUsage/STORAGE_BINDING}}</span>
```

Comment:
I feel like this header could use an update to clarify it's meaning relative to the next column.

```suggestion
            <th><span class=vertical>{{GPUStorageTextureAccess/"write-only"}}<br>
            {{GPUTextureUsage/STORAGE_BINDING}}</span>
```